documenters_aggregator/spiders/rta.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+import scrapy
+
+import re
+from datetime import datetime
+from pytz import timezone
+
+# The RTA's Board and other meetings are are displayed on their
+# website via an iframe from a different domain.
+class RtaSpider(scrapy.Spider):
+    name = 'rta'
+    allowed_domains = ['http://www.rtachicago.org', 'http://rtachicago.granicus.com']
+    start_urls = ['http://www.rtachicago.org/about-us/board-meetings']
+    domain_root = 'http://www.rtachicago.org'
+
+    def parse_iframe(self, response):
+        description = response.request.meta['description']
+        for item in response.css('#upcoming .row'):
+            yield {
+                '_type': 'event',
+                'id': self._parse_id(item),
+                'name': self._parse_name(item),
+                'description': description,
+                'classification': self._parse_classification(item),
+                'start_time': self._parse_start(item),
+                'end_time': self._parse_end(item),
+                'all_day': self._parse_all_day(item),
+                'status': self._parse_status(item),
+                'location': self._parse_location(item),
+            }
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows the `Open Civic Data
+        event standard <http://docs.opencivicdata.org/en/latest/data/event.html>`_.
+        """
+
+        description = response.css('.show_item_intro_text p::text').extract_first()
+
+        url = response.css('iframe::attr(src)').extract_first()
+
+        request = scrapy.Request(url, callback = self.parse_iframe)
+        request.meta['description'] = description
+
+        yield request
+
+    def _parse_id(self, item):
+        """
+        There is no publicly exposed ID.
+        """
+        return None
```

Comment:
I'm planning to generate an identifier from the date, time, and title, which will hopefully be unique enough.